### PR TITLE
update issue templates

### PR DESCRIPTION
**Pull request by KingOfTac**
- I removed the general and feature request templates because they are more fit for public facing projects where issues are created by external contributors and entities.
- The templates were replaced with `epic` and `story` templates which work better for internal product development.
- The `business request` template was renamed so that all of the the issue templates now map to a value for the `Type` field in our projects.
- The Treatment Suite project was also updated to deprecate the `Feature`, `Chore`, and `Fix` types and add the `Epic` and `Story` types.
- The deprecated values for the Type field will be removed in the near future once issues have been migrated or it has been determined that they are no longer needed for historical purposes.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/sow.yml
Comment (awentzel) [inline .github/ISSUE_TEMPLATE/sow.yml]: I think we should pluralize the formats in 'titles'. So, 'adds ..`. `updates ..`, `removes ..` This way when we automate the changelogs we can reuse these titles and it speaks directly to what a user gets from a feature, fix, etc.

File: .github/ISSUE_TEMPLATE/epic.md
Comment (awentzel) [inline .github/ISSUE_TEMPLATE/epic.md]: @bentomands what do you think would be the acceptance criteria for an 
Epic?

File: .github/ISSUE_TEMPLATE/sow.yml
Comment (awentzel) [inline .github/ISSUE_TEMPLATE/sow.yml:4]: Should a Statement of Work be filed as a stubbed in request? I'm sort of in the mind that any issue that gets created should be thoroughly and completely done. Hence, I wonder if having a default label for status should exist? Thoughts @bentomands @KingOfTac @cygleason

File: .github/ISSUE_TEMPLATE/story.md
Comment (awentzel) [inline .github/ISSUE_TEMPLATE/story.md]: "usually" or "always" part of a larger Epic. Is the hierarchical structure mandatory? I feel like it may need to be to reenforce the proper behavior.

File: .github/ISSUE_TEMPLATE/story.md
Comment (awentzel) [inline .github/ISSUE_TEMPLATE/story.md]: I'm so opinionated. ;-). I think 'fixes' would fall into 'updates [what] in [where]' Because I want it to be present tense to what the user receives as improvements not problems.  For example, "updates to allow for network success when requesting submission in change health'. or "updates to change primary color from red to blue".

Thus, instead of fixes would be 'removes'

File: .github/ISSUE_TEMPLATE/story.md
Comment (awentzel) [inline .github/ISSUE_TEMPLATE/story.md:10]: I do believe that Petrus and Nate should be on all Code Reviews for application stack changes. Barnett would be required on all DevOps and Data changes. But, this is just a note because even Petrus and Nate can benefit from code reviews and also demonstrate good behavior.

**Review by awentzel — COMMENTED** [review]
Opps. Forgot to push Review changes.

File: .github/ISSUE_TEMPLATE/epic.md
Comment (KingOfTac) [reply .github/ISSUE_TEMPLATE/epic.md]: Wouldn't the acceptance criteria change for each feature? For example adding something like passkeys to an existing authentication experience would have a specific criteria for acceptance, however adding something like metrics to a dashboard would have different criteria. Or am I wrong about what acceptance criteria is?

File: .github/ISSUE_TEMPLATE/sow.yml
Comment (KingOfTac) [reply .github/ISSUE_TEMPLATE/sow.yml:4]: I think since SOWs use the form, all of the information should already be filled out anyway so a status label probably isn't needed.

File: .github/ISSUE_TEMPLATE/story.md
Comment (KingOfTac) [reply .github/ISSUE_TEMPLATE/story.md]: I think you're right. I was thinking there may be some cases where a Story can be a standalone item, but that would really just be for engineering or GitHub related stuff in which case we could just create a separate template for those items that aren't user story related.

File: .github/ISSUE_TEMPLATE/story.md
Comment (KingOfTac) [reply .github/ISSUE_TEMPLATE/story.md:10]: This would be enforced by CODEOWNERS policies so a peer review task should be used for getting teammates eyes on each others code. It helps build a mentality of working within a team and also enables teammates to share their strengths with each other.

An example would be I could go implement a new FAST component for something and use `document.getElementById` to get references to elements in the component. While that works and is technically correct, you as a peer could review the PR and suggest replacing `document.getElementById` with FAST's `ref` directive which would be more succinct.

So each PR should have signoff from a code owner and another engineer. This process has a number of benefits, namely:
1. Encourages working as a team and preventing silos.
2. Enables teammates to share their experience with each other, improving the overall quality of each change to the codebase.
3. Helps free up leads and code owners time a bit more so that they don't get stuck in review hell.

File: .github/ISSUE_TEMPLATE/epic.md
Comment (KingOfTac) [inline .github/ISSUE_TEMPLATE/epic.md:4]: Not sure what to have as a title placeholder for Epics.

File: .github/ISSUE_TEMPLATE/epic.md
Comment (awentzel) [reply .github/ISSUE_TEMPLATE/epic.md]: Acceptance criteria in my mind is the same as "Definition of Done" which is the same as a standard list of things that must always be done. For example, if building a web form? Has validation been added? Has input formatting been applied? Has accessibility testing been performed? Has performance testing been performed? Have unit tests? ... you get the idea.

File: .github/ISSUE_TEMPLATE/story.md
Comment (awentzel) [reply .github/ISSUE_TEMPLATE/story.md]: After talking through this with Court, I think that Stories cannot be any larger than 1 day of effort or 7 hours of work. This allows for the tasks to be no longer than 2 hours in a task list. This ensures productivity and quickly knowing when things are off-track.  So, to answer your question, I would create more stories. Essentially, I don't want tasks to be converted to issues. That tells me it's not broken down enough in stories.

File: .github/ISSUE_TEMPLATE/epic.md
Comment (KingOfTac) [reply .github/ISSUE_TEMPLATE/epic.md]: I think I'll rename this section to definition of done

**Comment by KingOfTac** [comment]
An older but related PR to add a pull request template. https://github.com/trans-stat/.github/pull/6

I would like to also get feedback on this to get it to fit our new process.

File: .github/ISSUE_TEMPLATE/epic.md
Comment (awentzel) [reply .github/ISSUE_TEMPLATE/epic.md:4]: I think it's really just the feature. It took be "Adds inventory enhancements" "Updates inventory enhancements. etc.  so 'Adds [feature] enhancements' then the stories focus more on details?

**Review by awentzel — APPROVED** [review]

**Pull request merged**
